### PR TITLE
feat(@cypress/react): Make correct plugins for different adapters/bundlers

### PR DESCRIPTION
**Pull request by dmtrKovalenko**

### CODE REVIEW

File: npm/react/examples/nextjs/package.json [hunk 1]
```diff
@@ -6,20 +6,21 @@
     "build": "next build",
     "build:static": "next build && next out",
     "check-coverage": "check-coverage components/Search.jsx && check-coverage pages/index.js && check-coverage pages/router.js",
-    "cy:open": "node ../../../../scripts/cypress open",
+    "cy:open": "node ../../../../scripts/cypress open-ct",
     "dev": "next",
-    "only-cove red": "only-covered components/Search.jsx pages/index.js pages/router.js",
+    "only-covered": "only-covered components/Search.jsx pages/index.js pages/router.js",
```
Comment (elevatebart) [inline npm/react/examples/nextjs/package.json:11]: ?????

**Review by elevatebart — APPROVED** [review]
Overall one comment:
Since we are adding files in the filesystem of a user it would be valuable to add comments to the added files to explain what each file is responsible for.

Food for thoughts

File: npm/react/examples/nextjs/package.json (hunk 1 above)
Comment (dmtrKovalenko) [reply npm/react/examples/nextjs/package.json:11]: Just fixing script name

Comment (elevatebart) [reply npm/react/examples/nextjs/package.json:11]: I know, my question was how was this ever working ??

**Pull request merged**